BuildAndTest.proj

```diff
@@ -79,28 +81,41 @@
           SkipUnchangedFiles="true"
           UseHardlinksIfPossible="true" />
 
-    <MakeDir Directories="$(CoreClrTestDirectory)\xUnitResults" />
+    <MakeDir Directories="$(CoreClrTestDirectory)/xUnitResults" />
+
+    <!-- Publish binaries for CoreClr test -->
+    <MSBuild BuildInParallel="true"
+             Projects="src/Test/DeployCoreClrTestRuntime/DeployCoreClrTestRuntime.csproj"
+             Properties="RuntimeIdentifier=win7-x64;PublishDir=$(CoreClrTestDirectory)"
```

Comment:
The xplat builds currently use the makefile for testing, so this is never run outside of Windows. It may be worthwhile to consider unifying these soon.